fendermint/vm/message/src/conv/from_eth.rs

```diff
@@ -37,7 +37,12 @@ pub fn to_fvm_message(tx: &Eip1559TransactionRequest) -> anyhow::Result<Message>
     // The `from` of the transaction is inferred from the signature.
     // As long as the client and the server use the same hashing scheme,
     // this should be usable as a delegated address.
-    let from = to_fvm_address(tx.from.unwrap_or_default());
+    let from = if let Some(from) = tx.from {
```

Comment:
Why not use [`Option::map_or_else()`](https://doc.rust-lang.org/std/option/enum.Option.html#method.map_or_else)?

Reply:
just a preference, but I think `map_or` will fit nicely here. thx for the tip.